e2e/cypress/integration/messaging/emoji_followed_by_punctuation.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Pease change filename to `emoji_followed_by_punctuation_spec.js` since we're running [test to files](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json#L9) with `_spec` suffix only.

Reply:
I have changed it. :)